src/index.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './components/App/App';
+import './index.css';
 
 ReactDOM.render(
   <App />,
-  document.getElementById('root'),
+  global.document.getElementById('root'),
```

Comment:
never seen this before what is the distinction between global and document?

Reply:
It refers to the same object.  If global was omitted then during property lookup the prototype chain would eventually resolve to the global object.  global.document is just a more explicit way of referring to the object which doesn't anger ESLint.  Alternatively you could add a comment at the top of the file with the following:

`/* eslint-env browser */`

to tell ESLint to ignore browser global variables ie document